packages/react-core/src/next/components/DualListSelector/DualListSelectorList.tsx

```diff
@@ -0,0 +1,74 @@
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/react-styles/css/components/DualListSelector/dual-list-selector';
+import { DualListSelectorListItem } from './DualListSelectorListItem';
+import * as React from 'react';
+import { DualListSelectorListContext } from './DualListSelectorContext';
+
+/** Acts as the container for DualListSelectorListItem sub-components. */
+
+export interface DualListSelectorListProps extends React.HTMLProps<HTMLUListElement> {
+  /** Content rendered inside the dual list selector list. */
+  children?: React.ReactNode;
+}
+
+export const DualListSelectorList: React.FunctionComponent<DualListSelectorListProps> = ({
+  children,
+  ...props
+}: DualListSelectorListProps) => {
+  const {
+    setFocusedOption,
+    isTree,
+    ariaLabelledBy,
+    focusedOption,
+    displayOption,
+    selectedOptions,
+    id,
+    onOptionSelect,
+    options,
+    isDisabled
+  } = React.useContext(DualListSelectorListContext);
+
+  // only called when options are passed via options prop
```

Comment:
Is this function ever used? I don't think options are passed via the options prop anymore?

Reply:
yeah this is used for handling key actions instead of mouse in the dual list selector panes - i removed the confusing part of the comment to make it clearer